api/app/Console/Commands/SyncNewRodFields.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Enums\CandidateRemovalReason;
+use App\Enums\PoolCandidateStatus;
+use App\Models\PoolCandidate;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class SyncNewRodFields extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:sync-rod-fields';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Where the new RoD fields should be non-null, insert an appropriate value';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $dateNow = Carbon::now();
+        $placedStatuses = PoolCandidateStatus::placedGroup();
+        $removedStatuses = PoolCandidateStatus::removedGroup();
+        $finalDecisionStatuses = [
+            PoolCandidateStatus::QUALIFIED_AVAILABLE->name,
+            PoolCandidateStatus::QUALIFIED_UNAVAILABLE->name,
+            PoolCandidateStatus::QUALIFIED_WITHDREW->name,
+            PoolCandidateStatus::PLACED_CASUAL->name,
+            PoolCandidateStatus::PLACED_TERM->name,
+            PoolCandidateStatus::PLACED_INDETERMINATE->name,
```

Comment:
Oops, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/10150/commits/5486d6457f0e199fb2d5caafd18989c8d2ac94c9 then grouped up the statuses to avoid mishaps in https://github.com/GCTC-NTGC/gc-digital-talent/pull/10150/commits/54bd00f31810b49f109c3d6688c93520f180d2c9